### DISCUSSION TITLE
希望建个群！

### DISCUSSION THREAD

**Discussion by yiyungent** [Hoshikawa-Kaguya/Sora#79] [随便聊聊]
希望建个群！
QQ群或者 Tg 都可以，
每次都来这里好不方便，有时候只是小问题，而且到这里看的人太少，也不及时，谢谢，
当然我也知道最好到这反馈

**Comment by yiyungent** [comment]
https://github.com/yiyungent/QQBotHub/tree/main/plugins/SoraPlugin

最近在用这个包开发插件，有好多小问题想问下，希望建个群，谢谢

**Comment by Yukari316** [comment]
有群的，多看看代码就能发现
还有就是，入群条件也需要自己看清楚
因为之前总有人不看文档就来群里乱问，所以我也不会把群号在这里发出来
